File: 03-CPP/03-operadores/06ejercicioValorDentroRango.cpp
#include <iostream>
using namespace std;

int main()
{

    int min = 0;
    int max = 5;
    int valor;
    bool dentroRango;

    // Pedir valores al usuario
    cout << "Introduce un valor entre 0 - 5 " << endl;
    cin >> valor;

    // Validar si el dato está dentro del rango
    dentroRango = valor > min && valor <= max;
    cout << "Valor dentro de rango? " << dentroRango << endl;

    return 0;
}

File: 04-CSHARP/03-Operadores/06ejercicioValorDentroRango.cs

var min = 0;
var max = 5;
int valor;
bool dentroRango; 

// Solicitar valores al usuario.
Console.Write("Ingresa un valor entre 0 - 5");
valor = Convert.ToInt32(Console.ReadLine());

// Validar si el dato está dentro del rango. 
dentroRango = valor > min && valor <= max;
Console.WriteLine($"Valor dentro de rango? {dentroRango}"); 

